pkg/scale/errors.go

```diff
@@ -0,0 +1,10 @@
+// Copyright 2022 ChainSafe Systems (ON)
+// SPDX-License-Identifier: LGPL-3.0-only
+
+package scale
+
+import "errors"
+
+var (
+	ErrVaryingDataTypeValueNotSet = errors.New("VaryingDataTypeValue has not been set")
```

Comment:
```suggestion
	ErrVaryingDataTypeNotSet = errors.New("VaryingDataTypeValue has not been set")
```

Reply:
good call!

Reply:
I got quite confused with the the chicken or egg  debate.... is the varying data type not set or the varying data type value not set in the varying data type :rofl: Anyway that was a fun observation